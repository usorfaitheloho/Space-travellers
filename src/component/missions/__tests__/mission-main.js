import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../../../redux/configureStore';
import Mission from '../mission';

it('Mission renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Mission />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
