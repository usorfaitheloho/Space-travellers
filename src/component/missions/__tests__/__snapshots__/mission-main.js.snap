// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mission renders correctly 1`] = `
<div>
  <table>
    <thead>
      <tr>
        <th
          aria-label="Mission col"
          style={
            Object {
              "width": "15%",
            }
          }
        >
          Mission
        </th>
        <th
          aria-label="desc column"
        >
          Description
        </th>
        <th
          aria-label="status mtd"
          style={
            Object {
              "width": "15%",
            }
          }
        >
          Status
        </th>
        <th
          aria-label="action btn"
          style={
            Object {
              "width": "15%",
            }
          }
        />
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;
